lib/preprocessor.js

```diff
@@ -76,7 +76,6 @@ function createCoveragePreprocessor (logger, helper, basePath, reporters, covera
   return function (content, file, done) {
     log.debug('Processing "%s".', file.originalPath)
 
-    var jsPath = file.originalPath.replace(basePath + '/', './')
```

Comment:
I am sure someone had a reason they thought this was a good idea, but mucking about with istanbuls output breaks it for any downstream consumers that expect the output to conform to the spec (including istanbuls own command line tools). This makes `karma-coverage` brittle as its manipulating internal implementation details in istanbuls output and breaking encapsulation.